### ISSUE TITLE
KeyError: 'topic' in "HABApp/openhab/map_events.py" after upgrading OpenHAB to v3.3.0 stable

### ISSUE THREAD

**Issue by lorenzodeveloper** [spacemanspiff2007/HABApp#298]
Hi,
After I updated openhab to v3.3.0 I cannot make my rules to work again. There is only an error that is popping up every time in the logs:

```
2022-07-07 11:15:34.467 [ERROR] [HABApp                              ] - Error 'topic' in on_sse_event:
2022-07-07 11:15:34.468 [ERROR] [HABApp                              ] - File "/opt/habapp/lib/python3.7/site-packages/HABApp/openhab/connection_handler/sse_handler.py", line 25, in on_sse_event
2022-07-07 11:15:34.468 [ERROR] [HABApp                              ] -     22   def on_sse_event(event_dict: dict):
2022-07-07 11:15:34.468 [ERROR] [HABApp                              ] -     23       try:
2022-07-07 11:15:34.468 [ERROR] [HABApp                              ] -     24           # Lookup corresponding OpenHAB event
2022-07-07 11:15:34.468 [ERROR] [HABApp                              ] - --> 25           event = get_event(event_dict)
2022-07-07 11:15:34.468 [ERROR] [HABApp                              ] -     26   
2022-07-07 11:15:34.469 [ERROR] [HABApp                              ] -     ..................................................
2022-07-07 11:15:34.469 [ERROR] [HABApp                              ] -      event_dict = {'type': 'ALIVE'}
2022-07-07 11:15:34.469 [ERROR] [HABApp                              ] -     ..................................................
2022-07-07 11:15:34.469 [ERROR] [HABApp                              ] - 
2022-07-07 11:15:34.469 [ERROR] [HABApp                              ] - File "/opt/habapp/lib/python3.7/site-packages/HABApp/openhab/map_events.py", line 29, in get_event
2022-07-07 11:15:34.469 [ERROR] [HABApp                              ] -     27   def get_event(_in_dict: dict) -> OpenhabEvent:
2022-07-07 11:15:34.469 [ERROR] [HABApp                              ] -     28       event_type: str = _in_dict['type']
2022-07-07 11:15:34.469 [ERROR] [HABApp                              ] - --> 29       topic: str = _in_dict['topic']
2022-07-07 11:15:34.470 [ERROR] [HABApp                              ] -     30   
2022-07-07 11:15:34.470 [ERROR] [HABApp                              ] -     ..................................................
2022-07-07 11:15:34.470 [ERROR] [HABApp                              ] -      _in_dict = {'type': 'ALIVE'}
2022-07-07 11:15:34.470 [ERROR] [HABApp                              ] -      OpenhabEvent = <class 'HABApp.openhab.events.base_event.OpenhabEvent'>
2022-07-07 11:15:34.470 [ERROR] [HABApp                              ] -      event_type = 'ALIVE'
2022-07-07 11:15:34.470 [ERROR] [HABApp                              ] -     ..................................................
2022-07-07 11:15:34.470 [ERROR] [HABApp                              ] - 
2022-07-07 11:15:34.470 [ERROR] [HABApp                              ] - KeyError: 'topic'
```

## What I tried:

1. Updating the system (everything was up to date)
2. Updating habapp (but I'm already using the latest 0.31.2)
3. rebooting the system
4. restarting services (both openhab and habapp)

## System info
Release = Debian GNU/Linux 10 (buster) - openhabian
Kernel = Linux 5.10.103-v8+
Platform = Raspberry Pi 4 Model B Rev 1.4
OpenHAB version = 3.3.0 stable
HABApp version = 0.31.2 
Python version = 3.7.3

Is there anything I can do?
Thank you.

**Comment by spacemanspiff2007** [comment]
It's a new event that was introduced last minute into openHAB.
Unfortunately the only way to fix this issue is to revert back to openHAB 3.2.

If you are feeling adventurous you can try the beta which fixes this issue but introduces other breaking changes.
You find the beta thread in the openHAB forums under "apps & services" -> "3rd party"

**Comment by spacemanspiff2007** [comment]
Fixed with 1.0
